src/v/cluster/rm_stm.cc

```diff
@@ -1852,6 +1852,7 @@ ss::future<> rm_stm::do_try_abort_old_tx(model::producer_identity pid) {
           model::partition_id(0), pid, *tx_seq, _sync_timeout);
         if (r.ec == tx_errc::none) {
             if (r.commited) {
+                vlog(_ctx_log.trace, "RITA: commit on expire {} {}",pid, *tx_seq);
```

Comment:
what is RITA?

Reply:
oops, when I debug I put random words for grepping the log, RITA is one of the random words, updated the commit message

Reply:
gotcha. i do this too. i probably should write here in github what the words are that i use.